diff --git a/resources/com.sayonara-player.Sayonara.appdata.xml.in b/resources/com.sayonara-player.Sayonara.appdata.xml.in
index c1de7df8..31953010 100644
--- a/resources/com.sayonara-player.Sayonara.appdata.xml.in
+++ b/resources/com.sayonara-player.Sayonara.appdata.xml.in
@@ -12,7 +12,7 @@
 		<project_license>GPL-3.0+</project_license>
 		<releases>
 			<release version="@SAYONARA_FULL_VERSION@" date="@APPSTREAM_BUILD_DATE@">
-				<description>See release notes on https://sayonara-player.com/changelog</description>
+				<description><p>See release notes on https://sayonara-player.com/changelog</p></description>
 				<issues>
 					<issue url="https://gitlab.com/luciocarreras/sayonara-player">Search or file in issue reports at Gitlab</issue>
 				</issues>
@@ -28,15 +28,19 @@
 		<screenshots>
 			<screenshot type="default">
 				<image>https://sayonara-player.com/media/images/appdata/sayonara-appdata-01.png</image>
+				<caption xml:lang="en">Sayonara user interface</caption>
 			</screenshot>
 			<screenshot>
 				<image>https://sayonara-player.com/media/images/appdata/sayonara-appdata-02.png</image>
+				<caption xml:lang="en">Sayonara Equalizer</caption>
 			</screenshot>
 			<screenshot>
 				<image>https://sayonara-player.com/media/images/appdata/sayonara-appdata-03.png</image>
+				<caption xml:lang="en">Sayonara music collection</caption>
 			</screenshot>
 			<screenshot>
 				<image>https://sayonara-player.com/media/images/appdata/sayonara-appdata-04.png</image>
+				<caption xml:lang="en">Sayonara web radio</caption>
 			</screenshot>
 		</screenshots>
 		<url type="homepage">https://sayonara-player.com</url>
